bip-musig2-descriptors.mediawiki

```diff
@@ -0,0 +1,67 @@
+<pre>
+  BIP: musig2-descriptors
+  Layer: Applications
+  Title: musig() Descriptor Key Expression
+  Author: Ava Chow <me@achow101.com>
+  Comments-Summary: No comments yet.
+  Comments-URI: https://github.com/bitcoin/bips/wiki/Comments:BIP-musig2-descriptor
+  Status: Draft
+  Type: Informational
+  Created: 2024-01-15
+  License: BSD-2-Clause
+</pre>
+
+==Abstract==
+
+This document specifies a <tt>musig()</tt> key expression for output script descriptors.
```

Comment:
The `2` is not a version number. It refers to the number of rounds of communication required. If it were a version number, then this would be musig3 (or higher), as IIRC musig-dn was actually the second variant.

I chose to use `musig()` since MuSig2 is the only variant in use currently, and I expect it to be the predominant variant of MuSig. If there are other variants in the future, we can be more specific for those, and I doubt that they would be numbered in the same way, considering that the number is not a version number.